dojo/api_v2/views.py

```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
         # include current month
         months_between += 1
 
-        endpoint_monthly_counts = get_period_counts_legacy(findings.qs.order_by('numerical_severity'), findings.qs.order_by('numerical_severity'), None,
-                                                            months_between, start_date,
-                                                            relative_delta='months')
+        endpoint_monthly_counts = get_period_counts_legacy(
+            findings.qs.order_by("numerical_severity"),
+            findings.qs.order_by("numerical_severity"),
+            None,
+            months_between,
+            start_date,
+            relative_delta="months",
+        )
 
     elif type(obj).__name__ == "Product":
         product = obj
 
         report_name = "Product Report: " + str(product)
-        report_title = "Product Report"
-        report_subtitle = str(product)
-        findings = ReportFindingFilter(request.GET, product=product, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product=product)))
+        str(product)
+        findings = ReportFindingFilter(
+            request.GET,
+            product=product,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test__engagement__product=product)
+            ),
+        )
         ids = set(finding.id for finding in findings.qs)
-        engagements = Engagement.objects.filter(test__finding__id__in=ids).distinct()
+        engagements = Engagement.objects.filter(
+            test__finding__id__in=ids
+        ).distinct()
         tests = Test.objects.filter(finding__id__in=ids).distinct()
-        ids = get_endpoint_ids(Endpoint.objects.filter(product=product).distinct())
+        ids = get_endpoint_ids(
+            Endpoint.objects.filter(product=product).distinct()
+        )
         endpoints = Endpoint.objects.filter(id__in=ids)
 
     elif type(obj).__name__ == "Engagement":
         engagement = obj
-        findings = ReportFindingFilter(request.GET, engagement=engagement,
-                                       queryset=prefetch_related_findings_for_report(Finding.objects.filter(test__engagement=engagement)))
+        findings = ReportFindingFilter(
+            request.GET,
+            engagement=engagement,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test__engagement=engagement)
+            ),
+        )
         report_name = "Engagement Report: " + str(engagement)
 
-        report_title = "Engagement Report"
-        report_subtitle = str(engagement)
+        str(engagement)
```

Comment:
```suggestion

```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/commit/09e8746cdb2cce9fd1e93630862bc819b95e5c15